 <div style="display:inline-block; position:absolute; top:0; right:0; color: #230e3e; font:italic 1em serif;">$\LaTeX$<span style="display:inline-block; vertical-align:bottom">
    <a href="https://www.mathjax.org">
    <img style="height:1.5em" title="Powered by MathJax"
    src="https://www.mathjax.org/badge/mj_logo.png"
    border="0" alt="Powered by MathJax" >
</a></span></div>

<div style="display:inline-block; position:absolute; top:0; left:0; color: #230e3e; font:italic 1em serif;"><a href="https://www.julialang.org">
    <img style="height:2.25em; vertical-align:top" title="Powered by Julia"
    src="./figures/julia-logo-mask.png"
    border="0" alt="Powered by Julia" >
</a></div>
</a></div>$\require{cancel} \require{mhchem} \require{bbox} \require{color} \require{enclose} \require{AMSmath} \require{AMSsymbols}$

<header style="background-color:#e7e2ee; color:#461d7c; line-height:2em; padding-left:1em; padding-right:1em; bottom:0; float:bottom; text-align:right; float:right;">Moran, Shapiro, Boettner, Bailey, <i>Fundamentals of Engineering Thermodynamics</i>, 9th edition</header>

### problem 4.034

### Given
Steam enters the first-stage turbine shown in the figure at 40 bar and 500°C with a mass flow rate of 40,000 kg/hr. Steam exits the first-stage turbine at 20 bar and 400°C. The steam is then reheated at constant pressure to 500°C before entering the second-stage turbine. Steam leaves the second stage as saturated vapor at 0.6 bar. Assume steady state operation and ignore stray heat transfer and kinetic and potential energy effects. 

![4.034 diagram](./figures/EAT_14908854309950_8181991609077116.png)

### Find
  $\quad$(a) volumetric flow rate of the steam at the inlet to the first-stage turbine, in m³/min,   
  $\quad$(b) the rate of heat transfer to the steam flowing through the reheater, in kW, and   
  $\quad$(c) the total power produced by the two stages of the turbine, in kW.   

### Get Organized   

List your assumptions:   
1. Steady state operation  
2. No heat losses from any components  
3. No kinetic and potential energy effects.

Let's determine what information we need to find in order to answer the questions.  

#### (a) volumetric flow rate of the steam at the inlet to the first-stage turbine, 

$\quad$ We need $\dot V_1.$ We are given $\dot m.$ We know $\dot V_1 = \dot m v_1$   
$\quad$ We'll resolve State ⓵ to get $v_1.$


#### (b) the rate of heat transfer to the steam flowing through the reheater   
$\quad$ We apply the energy equation to the reheater. _Remember, the steam is flowing, so it carries internal energy and flow energy in/out of the system._  

\begin{equation*}
 \dot E_{in} - \dot E_{out} = \color{red}\cancelto{\text{0: steady state}}{\frac {\rm d}{\rm dt}E_{sys}} \\[6pt] 
 \dot Q_\mathit{reheater,in} + \dot m \left( u_2 + P_2v_2 \right) - \dot m \left( u_3 + P_3v_3 \right) = 0 \\[12pt]
 \dot Q_\mathit{reheater,in} = \dot m \left( h_3 - h_2 \right)   
\end{equation*}

$\quad$ We'll resolve States ⓶ and ⓷ to get $h_3, h_2.$  


#### (c) the total power produced by the two stages of the turbine   
$\quad$ We apply the energy equation to each turbine and sum the results.  

$$\dot W_{total,out} = \dot W_\textit{first stage, out} + \dot W_\textit{second stage, out}$$  

$\quad$ We'll start with first stage turbine. _Again, the steam is flowing, so it carries internal energy and flow energy in/out of the system._   

\begin{equation*}
 \dot E_{in} - \dot E_{out} = \color{red}\cancelto{\text{0: steady state}}{\frac {\rm d}{\rm dt}E_{sys}} \\[6pt] 
 \dot m \left( u_1 + P_1v_1 \right) - \dot W_\textit{first stage, out} - \dot m \left( u_2 + P_2v_2 \right) = 0 \\[12pt]
 \dot W_\textit{first stage, out} = \dot m \left( h_1 - h_2 \right)   
\end{equation*}

$\quad$ Likewise, 
\begin{equation*}
 \dot W_\textit{second stage, out} = \dot m \left( h_3 - h_4 \right)  \\[15pt]
\end{equation*}

$\quad$ Finally, 
\begin{align*}
 \dot W_{total,out} 
     &= \dot m \left( h_1 - h_2 \right) + \dot m \left( h_3 - h_4 \right) \\ \\
\end{align*}  

$\quad$ Overall, the properties we need are $v_1,\ h_1,\ h_2,\ h_3,\ h_4.$

### Attack 

 Today, we'll use Julia, with _Unitful.jl_, a package for calculation with physical units.

In [1]:
using Unitful       # Unitful units package
const kJ = u"kJ"    # make shortcuts to Unitful units constructor
const kg = u"kg"
const m = u"m"
const kW = u"kW"
const minute = u"minute"
const hr = u"hr";

#### (a) volumetric flow rate of the steam at the inlet to the first-stage turbine, 

$\quad \dot V_1 = \dot m v_1$   

$\quad \left.
\begin{alignat*}{1}
P_1 &= 40\ \textsf{bar} \\
T_1 &= 500\ \textsf{℃} 
\end{alignat*}
\right\}
\rightarrow
v_1 = 0.08643\, \mathrm{m^3/kg}, \quad h_1 = 3445.3\, \mathrm{kJ/kg}
$  

$\quad$ _We get $h_1$ while we're in the table, knowing we need it later._

In [2]:
ṁ = 40000kg/hr              # given
h₁ = 3445.3kJ/kg            # table 
v₁ = 0.08643m^3/kg          # table

V̇₁ = ṁ * v₁ |> m^3/minute

57.620000000000005 m^3 minute^-1

$\color{red} \dot V_1 = 57.62\ \rm m^3/min$

<p style = " background-image:  url(./figures/bacgroundTiles.png); line-height: 0.35em" >&nbsp;</p>

#### (b) the rate of heat transfer to the steam flowing through the reheater   
$\quad \dot Q_\mathit{reheater,in} = \dot m \left( h_3 - h_2 \right) $

$\quad \left.
\begin{alignat*}{1}
P_3 &= 20\ \textsf{bar} \\
T_3 &= 500\ \textsf{℃} 
\end{alignat*}
\right\}
\rightarrow
 h_3 = 3467.6\, \mathrm{kJ/kg}
$
$\quad \left.
\begin{alignat*}{1}
P_2 &= 20\ \textsf{bar} \\
T_2 &= 400\ \textsf{℃} 
\end{alignat*}
\right\}
\rightarrow
 h_2 = 3247.6\, \mathrm{kJ/kg}
$ 

In [3]:
h₃ = 3467.6kJ/kg            # table 
h₂ = 3247.6kJ/kg            # table 

Q̇_reheater_in = ṁ * (h₃ - h₂) |> kW

2444.4444444444443 kW

$\color{red} \dot Q_\mathit{reheater,in} = 2444.44\, \rm kW$

<p style = " background-image:  url(./figures/bacgroundTiles.png); line-height: 0.35em" >&nbsp;</p>

#### (c) the total power produced by the two stages of the turbine   
$\quad \dot W_{total,out} = \dot m \left( h_1 - h_2 \right) + \dot m \left( h_3 - h_4 \right) $

$\quad \left.
\begin{array}{l}
P_4 = 0.6\ \textsf{bar} \\
\text{saturated vapor}
\end{array}
\right\}
\rightarrow
 h_4 = 2653.5\, \mathrm{kJ/kg}
$

In [4]:
h₄ = 2653.5kJ/kg            # table 

Ẇ_total_out = ṁ * (h₁ - h₂ + h₃ - h₄) |> kW

11242.222222222224 kW

$\color{red} \dot W_{total,out} = 11242.22\ \rm kW$

<footer style="background-color: #461d7c; color:rgba(253,208,35,0.35); line-height:2em; padding-left:1em; padding-right:0.5em; bottom:0; float:bottom; text-align:right;"><i>ψᵟ</i></footer>